examples/object-type/object_type.description

```diff
@@ -1,4 +1,4 @@
-// Objects are a behaviourial type. An object type is a combination of public and module-level visible fields along with a
+// Objects are a behavioural type. An object type is a combination of public and module-level visible fields along with a
```

Comment:
This should actually be "behavioral". American spelling is usually used in the site.

Reply:
Done via b246ead103557750b09882715bccbbf7f0d4c5e4